vmdb/app/models/job.rb

```diff
@@ -55,7 +55,7 @@ def check_active_on_destroy
 
   def self.agent_state_update_queue(jobid, state, message=nil)
     begin
-      job = Job.find(:first, :conditions => ["guid = ?", jobid], :select => "id, state, guid")
+      job = Job.where("guid = ?", jobid).select("id, state, guid").first
```

Comment:
Need an Array for `.select`.

ASIDE: @tenderlove Is there a reason that AR::Relation#select requires an Array and not just splatted args?  .pluck, .order, and others take splatted args, and it's confusing to have select act differently.  It's so easy to make a mistake.